README.md

```diff
@@ -58,20 +58,20 @@ The spec supports request types of “erasure”, "access" and “portability”
 ## Endpoints
 This is an overview of available HTTP methods for communicating between Controllers and Processors. The following endpoints should be provided by the Processor (to receive requests from the Controller).
 
-Restful API endpoints for the resource “opengdpr_requests”:
+Restful API endpoints for the resource “OpenCompliance_requests”:
 
 | HTTP Method | Path | Description | Supported? |
 | --- | --- | --- | --- |
-| POST | opengdpr_requests/ | Create a new OpenGDPR request | Yes |
-| GET | opengdpr_requests/{RequestId} | Retrieve status of a single OpenGDPR request | Yes |
-| PUT | opengdpr_requests/{RequestId} | - | No, requests cannot be updated after being created |
-| DELETE | opengdpr_requests/{RequestId} | Cancel an OpenGDPR request | Yes, cancellation is valid in status “pending” only |
+| POST | OpenCompliance_requests/ | Create a new OpenCompliance request | Yes |
```

Comment:
Going with a lowercase route will eliminate confusion about case sensitivity and be more similar to the "voice" v1 has.

It might be worth considering a more generic entity name here such as `subject_requests` if the industry is converging on a single vocabulary. I'm posing a question more than making a suggestion. Thoughts?